docs/virtual_machines/interfaces_and_networks.md

```diff
@@ -1079,17 +1080,13 @@ There are two known CNI plugins that support mac-spoof-check:
 
 - [sriov-cni](https://github.com/openshift/sriov-cni):
   Through the `spoofchk` parameter .
-- cnv-bridge: Through the `macspoofchk`.
-
-> **Note:** `cnv-bridge` is provided by
-  [CNAO](https://github.com/kubevirt/cluster-network-addons-operator).
-  The [bridge-cni](https://github.com/containernetworking/plugins) is planned
-  to support the `macspoofchk` options as well.
+- [bridge-cni](https://github.com/containernetworking/plugins):
+  Through the `macspoofchk`.
```

Comment:
What is this?

Reply:
But this line I commented on is left here.

Reply:
OK, now I understand. I think you just need to sync it with the previous text. i.e. `Through the macspoofchk parameter`.

Reply:
Thanks for the heads up, fixed.